### ISSUE TITLE
[Plugin Request]

### ISSUE THREAD

**Issue by thedountman** [TheLazySquid/Gimloader#5]
### Describe the plugin you would like to see added

just an addon to the 2d tas, is there a way to select multiple movement actions and change their angle? if not could you add it? thanks!

**Comment by TheLazySquid** [comment]
Yep! It's not well communicated but dragging the arrow on the right will copy the angle. Also, be warned that 2D TAS is still in an early state, and it can be pretty easy to desync the playback.

**Comment by thedountman** [comment]
Thanks so much, apparently u can buy the same upgrade a maximum of 13
times, leading to a 39 times speed increase (or energy reduction, which if
u buy enough gives u energy for walking, and energy per question thats an
insane amount). I guess its cool :)

**Comment by TheLazySquid** [comment]
Uh oh, that's definitely a bug. Upgrades should only be purchasable once.

**Comment by thedountman** [comment]
I mean its cool cus zoom but yeah a bug nonetheless, also thanks for
reading the thing I sent!
